test/test_load_ref_schema.rb

```diff
@@ -12,29 +12,19 @@ def load_other_schema
           }
         }
       },
-      Addressable::URI.parse("http://example.com/schema#")
+      Addressable::URI.parse(url)
```

Comment:
It doesn't seem like it. `JSON::Schema` expects 2 arguments, one is the schema as a hash, the second is the `schema_url` as a `URI`-like object.